.github/terraform/hazelcast-client.yaml

```diff
@@ -0,0 +1,5 @@
+hazelcast-client:
+  network:
+    gcp:
```

Comment:
If we don't give those parameters, discovery plugin will find the zone and region from instance api. It is not needed but we can add it.

`region: a region where the plugin looks for instances; if not set, the zones property is used; if it and zones property not set, all zones of the current region are used`

This is the explanation of  `region` property written in README of `hazelcast-gcp`.

Reply:
OK, I see. But trying all zones just because the config is not set seems poor to me. I'm not sure how it affects the discovery time of other members tho. Setting via sed similar to aws tests would be fine here. However, changing or not is up to you then.